keyboards/redox/proton_c/config.h

```diff
@@ -0,0 +1,84 @@
+// Copyright 2022 xerootg
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4D44 // "MD"
+#define PRODUCT_ID      0x5244 // "RD"
+#define DEVICE_VER      0x0101 // added 1 for the rev1 + proton c
+#define MANUFACTURER    Falbatech
+#define PRODUCT         The Redox Keyboard
+
+/* key matrix size */
+// Rows are doubled-up
+#define MATRIX_ROWS 10
+#define MATRIX_COLS 7
+
+// wiring of each half
+#define MATRIX_ROW_PINS { B5, B3, B2, B1, B0 }
+#define MATRIX_COL_PINS { A1, A0, B8, B13, B14, B15, B9 }
+// #define MATRIX_COL_PINS { B6, B2, B3, B1, F7, F6, F5} //uncomment this line and comment line above if you need to reverse left-to-right key order
```

Comment:
What do you mean by fix? It is accurate.

Reply:
I suppose that the intent of that line is to have the same pins as above, but in the reverse order; because you are not using `CONVERT_TO_PROTON_C = yes`, you changed the pin names above, but the commented out example should get the same change (otherwise it is just confusing).

Reply:
OOOHH Ok, pulling it. It was not clear that the commented out ```MATRIX_COL_PINS``` line is what you were talking to ;)